docker.go

```diff
@@ -478,6 +478,13 @@ func (c *DockerContainer) NetworkAliases(ctx context.Context) (map[string][]stri
 	return a, nil
 }
 
+// Exec executes a command in the current container.
+// It returns the exit status of the executed command, an `io.Reader` containing the combined
+// stdout and stderr, and any encountered error. Note that reading directly from the `io.Reader`
+// may result in unexpected bytes due to custom stream multiplexing headers.
+// To separate the stdout and stderr from `io.Reader` and interpret these headers properly,
+// `stdcopy.StdCopy()` from the Docker API should be used.
+// More information about this can be found at: https://godoc.org/github.com/docker/docker/pkg/stdcopy
```

Comment:
Thanks for contributing this. It probably caused some pain while using the method and I'm very thankful you decide to add it for others not to suffer the same 🙇 

We have the `Multiplexed` process option that separates the stdout and stderr using the stdcopy API, so I'd change this comment to use it.

I love having docs for this method, but I'd enforce the usage of the right process option instead. Wdyt?

Reply:
I've tested the `Multiplexed` option and found that it doesn't combine the output as expected, similar to how it's done in [`cmd.CombinedOutput()`](https://pkg.go.dev/os/exec#Cmd.CombinedOutput). Specifically, in the scenario where stderr contains data, it's preferred over stdout:
 
https://github.com/testcontainers/testcontainers-go/blob/32f372c44be48f9a4b271555f5e6a1e20238cbaf/exec/processor.go#L85-L89

It seems there might be a need to change the behavior of the `Multiplexed` option or introduce a new option like `CombinedOutput`.

What do you think?

Reply:
I like the idea: having different options or fixing the existing one makes sense to me.

Would you like to work on that?

Reply:
I can work on a new option called `CombinedOutput`, or I can investigate fixing the behavior of the existing `Multiplexed`. However, my concern is that changing the behavior of something existing might break compatibility with code that relies on the current behavior (preferring stderr if it contains data).

Let me know your thoughts on how you'd like to proceed.

Reply:
We can parametrize the Multiplexed option. It would mean a breaking change, but for the sake of consistency, it could be seen as needed.